config/settings_schema.json

```diff
@@ -24,6 +24,25 @@
       }
     ]
   },
+  {
+    "name": "Metafields",
+    "settings":[
+      {
+        "type": "text",
+        "id": "metafield_buckets",
+        "label": "Metafield Buckets",
+        "default": "https://s3.amazonaws.com/metafields.blendtec.com",
+        "info": "The Bucket where metafields stores the images"
+      },
```

Comment:
why do we need both of these? they point to the same place, right? just one allows https. 
https://s3.amazonaws.com/metafields.blendtec.com/blendtec/Classic575_Interface-1518620839868.png
http://metafields.blendtec.com.s3.amazonaws.com/blendtec/Classic575_Interface-1518620839868.png